packages/backend/src/tokens/request.ts

```diff
@@ -37,9 +37,59 @@ export type OptionalVerifyTokenOptions = Partial<
   Pick<VerifyTokenOptions, 'authorizedParties' | 'clockSkewInSeconds' | 'jwksCacheTtlInMs' | 'skipJwksCache' | 'jwtKey'>
 >;
 
-export type AuthenticateRequestOptions = RequiredVerifyTokenOptions &
+type PublicKeys =
+  | {
+      publishableKey: string;
+      /**
+       * @deprecated Use `publishableKey` instead.
+       */
+      frontendApi: never;
+    }
+  | {
+      publishableKey: never;
+      /**
+       * @deprecated Use `publishableKey` instead.
+       */
+      frontendApi: string;
+    }
+  | {
+      publishableKey: string;
+      /**
+       * @deprecated Use `publishableKey` instead.
+       */
+      frontendApi: string;
```

Comment:
Why can we have both?

Reply:
You have wayy more context that me here so please ignore if it doesn't make sense, however, the usual way to write this would be:
```ts
type ExclusiveAOrB = {
 a: string; 
 b?: never; 
} | {
 a?: never;
 b: string;
}
```
(notice the `?`)

Reply:
> Why can we have both?

@nikosdouvlis Isn't this a breaking change (on the type-level) of anyone passing both?